src/items/itemsTable.component.tsx

```diff
@@ -147,6 +151,10 @@ export function ItemsTable(props: ItemTableProps) {
     'create' | 'duplicate' | 'edit'
   >('create');
 
+  const [openDialogAsPrivilegedUser, setOpenDialogAsPrivilegedUser] =
+    React.useState<boolean>(false);
```

Comment:
```suggestion
  const [openDialogAsPrivilegedUser, setOpenDialogAsPrivilegedUser] =
    React.useState<boolean>(false);
```

The name of this is confusing, as this state is not opening the dialog, it set the dialog to use privileged version. Can you change this please